test/site/get-site-status-by-exporter-name.api-test.ts

```diff
@@ -0,0 +1,129 @@
+import { IncorrectAuthArg, withClientAuthenticationTests } from '@ukef-test/common-tests/client-authentication-api-tests';
+import { withCommonGraphExceptionHandlingTests } from '@ukef-test/common-tests/common-graph-exception-handling-api-tests';
+import { Api } from '@ukef-test/support/api';
+import { getSiteStatusByExporterNameGenerator } from '@ukef-test/support/generator/get-site-status-by-exporter-name-generator';
+import { RandomValueGenerator } from '@ukef-test/support/generator/random-value-generator';
+import { MockGraphClientService } from '@ukef-test/support/mocks/graph-client.service.mock';
+import { resetAllWhenMocks } from 'jest-when';
+
+describe('getSiteStatusByExporterName', () => {
+  const valueGenerator = new RandomValueGenerator();
+  const {
+    siteStatusByExporterNameQueryDto,
+    graphServiceGetParams: { path, expand, filter },
+    graphGetSiteStatusResponseDto,
+  } = new getSiteStatusByExporterNameGenerator(valueGenerator).generate({
+    numberToGenerate: 1,
+  });
+
+  let api: Api;
+  let mockGraphClientService: MockGraphClientService;
+
+  beforeAll(async () => {
+    ({ api, mockGraphClientService } = await Api.create());
+  });
+
+  beforeEach(() => {
+    jest.resetAllMocks();
+    resetAllWhenMocks();
+  });
+
+  afterAll(async () => {
+    await api.destroy();
+  });
+
+  withClientAuthenticationTests({
+    givenTheRequestWouldOtherwiseSucceed: () => {
+      mockGraphClientService
+        .mockSuccessfulGraphApiCallWithPath(path)
+        .mockSuccessfulExpandCallWithExpandString(expand)
+        .mockSuccessfulFilterCallWithFilterString(filter)
+        .mockSuccessfulGraphGetCall(graphGetSiteStatusResponseDto);
+    },
+    makeRequestWithoutAuth: (incorrectAuth?: IncorrectAuthArg) =>
+      api.getWithoutAuth(`/api/v1/sites?exporterName=${siteStatusByExporterNameQueryDto.exporterName}`, incorrectAuth?.headerName, incorrectAuth?.headerValue),
+  });
+
+  withCommonGraphExceptionHandlingTests({
+    givenRequestWouldOtherwiseSucceed: () => {
+      mockGraphClientService
+        .mockSuccessfulGraphApiCallWithPath(path)
+        .mockSuccessfulExpandCallWithExpandString(expand)
+        .mockSuccessfulFilterCallWithFilterString(filter);
+    },
+    givenGraphServiceCallWillThrowError: (error: Error) => {
+      mockGraphClientService.mockUnsuccessfulGraphGetCall(error);
+    },
+    makeRequest: () => api.get(`/api/v1/sites?exporterName=${siteStatusByExporterNameQueryDto.exporterName}`),
```

Comment:
We use this string interpolation a few times in this file, can we make a constant or function for it please?